module/Order/src/Order/Service/ItemService.php

```diff
@@ -14,6 +14,8 @@
 
 class ItemService extends AbstractService
 {
+    const PAGINATION_MAX_ROWS = 5;
```

Comment:
Yes, we have that in `Foundation\AbstractRepository` but this is kind of specific for Item entity only so I kept it here.